README.md

```diff
@@ -157,6 +175,20 @@ The optional configuration JSON file defaults to the `convertToStdVCF.json` resi
 
   * patch: Remove all code related to PyPy and hts-python (including `copysam.py` and `PYPY_OR_PYTHON_BINARY`)
 
+
+* 3.0.0
+
+  * Major
+    * Support for hg38/GRCh38 reference genome and variant calling from ALT and HLA contigs.
+  * Minor
+    * For hg38: Removing mappability and repeat elements' annotations from penalty calculations.
+    * skipREMAP: Option to remove repeat elements and mappability from confidence annotations in hg19.
+    * Removing EVS And ExAC AF from the annotations and no-control workflow filtering
+    * Support for variant calling from CRAM files
+    * Bug fix: Removing "quote" around the raw filter option `<RAW_SNV_FILTER_OPTIONS>`
```

Comment:
Right. But this is a technical/code description. What was the effect of the bug to the user? (probably that multiple `RAW_SNV_FILTER_OPTIONS` could not be used).